ocamlformat-help.txt

```diff
@@ -297,13 +304,32 @@ OPTIONS
            Read .ocp-indent configuration files. match-clause sets
            cases-exp-indent to the same value.
 
-       -p {default|compact|sparse|janestreet},
-       --profile={default|compact|sparse|janestreet}
-           Preset profiles which set all options, overriding lower priority
-           configuration. default sets each option to its default value.
-           compact sets options for a generally compact code style. sparse
-           sets options for a generally sparse code style. janestreet is the
-           profile used at JaneStreet. 
+       -p {ocamlformat|compact|sparse|conventional|janestreet|default},
+       --profile={ocamlformat|compact|sparse|conventional|janestreet|default}
+           Select a preset profile which sets all options, overriding lower
+           priority configuration. The ocamlformat profile aims to take
+           advantage of the strengths of a parsetree-based auto-formatter,
+           and to limit the consequences of the weaknesses imposed by the
+           current implementation. This is a style which optimizes for what
+           the formatter can do best, rather than to match the style of any
+           existing code. General guidelines that have directed the design
+           include: Legibility, in the sense of making it as hard as possible
+           for quick visual parsing to give the wrong interpretation, is of
+           highest priority; Whenever possible the high-level structure of
+           the code should be obvious by looking only at the left margin, in
+           particular, it should not be necessary to visually jump from left
+           to right hunting for critical keywords, tokens, etc; All else
+           equal compact code is preferred as reading without scrolling is
+           easier, so indentation or white space is avoided unless it helps
+           legibility; Attention has been given to making some syntactic
+           gotchas visually obvious. The compact profile is similar to
+           ocamlformat but opts for a generally more compact code style. The
+           sparse profile is similar to ocamlformat but opts for a generally
+           more sparse code style. The conventional profile aims to be as
+           familiar and "conventional" appearing as the available options
+           allow. The janestreet profile is used at Jane Street. The
+           "default" profile is ocamlformat with break-cases=fit. default is
+           deprecated and will be removed in version 0.10. 
```

Comment:
It would be nice if there was a way to get more structure in this, but I don't see how to convince cmdliner to do so.

Reply:
I didn't find a way to add some linebreaks.